keps/sig-storage/1845-prioritization-on-volume-capacity/README.md

```diff
@@ -0,0 +1,505 @@
+checklist items _must_ be updated for the enhancement to be released.
+-->
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [x] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+With [volume topology-aware
+scheduling](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/volume-topology-scheduling.md),
+scheduler will filter nodes considering volume topology constraints. Unbound
+PVCs will be bound with smallest fit PVs in a topology, however scheduler does
+not try to prioritize nodes in different topologies based on volume capacity.
+This leads to waste of resources which we should avoid.
+
+This KEP continues the work proposed [here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/volume-topology-scheduling.md#priority).
+
+## Motivation
+
+Because we didn't take volume capacity into account when scheduling a pod that
+can run in multiple topologies. A large PV may be used by a PVC that has very
+small capacity request, even if there are suitable small PVs in other
+topologies. PVCs with large capacity request may not find feasible PVs to use,
+if too many large PVs are consumed by PVCs with small capacity request. 
+
+Like CPU, Memory resources, scheduler should take volume capacity into account
+in scheduling pods to ensure balanced resource usage.
+
+### Goals
+
+- Nodes of topology in which volumes with close capacity are preferred in
+  scheduling
+- Support arbitrary PV topology domain (e.g. hostname, zone, rack)
+
+### Non-Goals
+
+- Prioritizing nodes based on total available capacity of static provisioned
+  PVs
+- Prioritizing nodes based on total available number of static provisioned
+  PVs
+- Prioritizing nodes based on total available capacity for dynamic
```

Comment:
However, this is necessary when storage capacity tracking is ready. Maybe a new KEP or an update to Storage Capacity Tracking in the future?

xref: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1472-storage-capacity-tracking/README.md#prioritization-of-nodes